### PR TITLE
docs(sbb-action-group): add button-size small

### LINKED ISSUE #2583: lean: action-group: size S
https://www.figma.com/file/9r6xSfNmEfCFxl1yFYedrj/Lyne-Components?type=design&node-id=29287%3A1077432&mode=design&t=6s3eVNfR5eROI3dH-1

### PR DESCRIPTION
**Pull request by DavideMininni-Fincons**
## Preflight Checklist

- [x] I have read the [Contributing Guidelines](https://github.com/lyne-design-system/lyne-components/blob/main/CONTRIBUTING.md) for this project.
- [x] I agree to follow the [Code of Conduct](https://github.com/lyne-design-system/lyne-components/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [x] I have searched the [pull request tracker](https://github.com/lyne-design-system/lyne-components/pulls) for a Pull Request (PR) that matches the one I want to submit, without success.

## Issue

This PR Closes #2583 

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

See [Review Guidelines](../REVIEW.md) for more information what is checked during review process.

## Changes

Changes in this pull request:

- add stories with button-size='s' in action-group stories

## Browsers

I tested the build on the following browsers:

- [ ] Firefox Desktop
- [ ] Chrome Desktop
- [ ] Edge Desktop
- [ ] Safari Desktop
- [ ] Chrome Mobile
- [ ] Safari Mobile

## Screen readers

I tested the build on the following browsers:

- [ ] JAWS Firefox Desktop
- [ ] JAWS Chrome Desktop
- [ ] NVDA Firefox Desktop
- [ ] NVDA Chrome Desktop
- [ ] VoiceOver Safari Desktop
- [ ] VoiceOver Chrome Desktop
- [ ] VoiceOver Safari Mobile
- [ ] Android Accessibility Suite Chrome Mobile

## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by jeripeierSBB — APPROVED** [review]
LGTM, maybe shorten stories

File: src/components/action-group/action-group.stories.ts [hunk 1]
```diff
@@ -266,6 +276,78 @@ export const VerticalToHorizontal3_0_0: StoryObj = {
   args: { ...basicArgsVertical, 'horizontal-from': 'medium' },
 };
 
+export const HorizontalAllocationSizeS3_0_0: StoryObj = {
```
[the remark this answers is not in the document]
Comment (jeripeierSBB) [reply src/components/action-group/action-group.stories.ts:279]: or just having one additional story?

[the remark this answers is not in the document]
Comment (DavideMininni-Fincons) [reply src/components/action-group/action-group.stories.ts:281]: Just left one story for vertical and one for horizontal

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**